Sources/NIOCore/ChannelHandler.swift

```diff
@@ -343,3 +343,19 @@ extension RemovableChannelHandler {
         context.leavePipeline(removalToken: removalToken)
     }
 }
+
+/// A `OutboundBufferedBytesAuditableChannelHandler` is a `ChannelHandler` that 
+/// audits and reports the number of bytes buffered for outbound direction.
+public protocol OutboundBufferedBytesAuditableChannelHandler {
```

Comment:
Per our API evolution rules, these will need to be prefixed with `NIO`: 

```suggestion
public protocol NIOOutboundBufferedBytesAuditableChannelHandler {
```